### PR TITLE
 ANDROID: cpufreq_stat: add per task/uid/freq stats

### PR DESCRIPTION
**Pull request by ygorigor**

### CODE REVIEW

**Review by nichcream — APPROVED** [review]

File: include/linux/cpufreq.h [hunk 1]
```diff
@@ -651,6 +651,18 @@ int cpufreq_generic_init(struct cpufreq_policy *policy,
  *********************************************************************/
 
 void acct_update_power(struct task_struct *p, cputime_t cputime);
+void cpufreq_task_stats_init(struct task_struct *p);
```
Comment (nichcream) [inline include/linux/cpufreq.h:654]: #ifdef CONFIG_CPU_FREQ_STAT?

**Comment by nichcream** [comment]
Might want to fix 3cefad1 and pull request again ^^

File: include/linux/cpufreq.h (hunk 1 above)
Comment (ygorigor) [reply include/linux/cpufreq.h:654]: Yes, my mistake. Fixed https://github.com/ygorigor/wingtech_kernel_msm8937/commit/298a16e78cbd495f49f0828dd1eec053597c346a

**Comment by ygorigor** [comment]
I'll try to compile and if is all ok I'll do a pull request

**Pull request merged**